docs/Drawer.md

```diff
@@ -78,34 +82,46 @@ import DrawerBody from '@pxblue/react-components/core/Drawer';
+| fontColor             | The color used for the text                                      | `string`          | no       |         |   
+| hidePadding           | Whether to hide the paddings reserved for menu item icons        | `boolean`         | no       |         | 
+| iconColor             | The color used for the icon                                      | `string`          | no       |         |   
+| items                 | List of navigation items to render                               | `NavItem[]`       | yes      |         | 
+| nestedDivider         | Whether to show a line between nested menu items                 | `boolean`         | no       | false   |    
+| onSelect              | Function to execute when clicking on any menu item               | `function`        | no       |         |
+| open                  | When false, group header will be hidden (like a collapsed state) | `function`        | no       |         |
+| ripple                | Whether to apply ripple effect on all the items                  | `boolean`         | no       | true    |
+| title                 | Text to display in the group header                              | `string`          | no       |         |  
+| titleColor            | The color used for the title                                     | `string`          | no       |         | 
+
+#### NavItem Object
+The `items` prop of the `DrawerNavGroup` takes a list of items with the following structure (most of these properties are inherited from `<InfoListItem/>`).
+
+| Attribute       | Description                                                                                                    | Type               | Required | Default                      |
+|-----------------|----------------------------------------------------------------------------------------------------------------|--------------------|----------|------------------------------|
+| chevron         | Show chevron icon to the right. Override by `rightComponent`                                                   | `boolean`          | no       | false                        |  
+| divider         | Show a divider line below the top-level item                                                                   | `boolean`          | no       | true                         | 
+| icon            | A component to render for the icon, not allowed for nested items                                               | `React.Component`  | no       |                              |    
+| onClick         | A function to execute when clicked                                                                             | `function`         | no       |                              |    
+| onClickIcon     | A function to execute when clicked on the rightComponent                                                       | `function`         | no       |                              |    
```

Comment:
If the user passes in the right component, don't they control what happens when it is clicked?